db/changes_view.go

```diff
@@ -89,7 +89,10 @@ func nextChannelQueryEntry(results sgbucket.QueryResultIterator, collectionID ui
 
 // Queries the 'channels' view to get a range of sequences of a single channel as LogEntries.
 func (dbc *DatabaseContext) getChangesInChannelFromQuery(ctx context.Context, channel channels.ID, startSeq, endSeq uint64, limit int, activeOnly bool) (LogEntries, error) {
```

Comment:
I'd err on the side of removing this method entirely given it's usage. All of the callers are tests, and have a collection available:
- `DatabaseContext.ChannelViewForTest`
- `TestQueryChannelsActiveOnlyWithLimit`
- `TestChannelView`